### DISCUSSION TITLE
Allow specified actions and reusable workflows not accessible from private repository

### DISCUSSION THREAD

**Discussion by brandonfl** [community/community#30050] [Actions]
Hello, 

Is that wanted that the feature that allow use to select specifieed actions into actions permissions are only available for private repositories ?

Here is an example : 

- Public repository  

![public](https://user-images.githubusercontent.com/25805069/185590212-41e0e521-48c1-49d7-babe-8c8e023e48fc.png)

- Private repository 

![private-2](https://user-images.githubusercontent.com/25805069/185590976-d2451a6a-abef-42fe-8d59-5f26ea775c5a.png)

I dont understand why this feature is not present into private repository :thinking: 

Thanks in advance, 
BrandonFL

**Comment by airtower-luna** [comment]
There are a bunch of features that aren't available to private repositories on free accounts. I'm pretty sure it's simply a sales decision, to encourage people to buy paid services. :sweat_smile:

**Reply by brandonfl** [reply]
I have a PRO account and not a free one :yum: 

![Capture d’écran de 2022-08-19 13-56-44](https://user-images.githubusercontent.com/25805069/185613431-e593fce2-9eb8-4245-9892-84e8e39e7de2.png)

**Reply by airtower-luna** [reply]
Huh, that baffled me, but it's [actually documented behavior](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-select-actions-and-reusable-workflows-to-run):

> **Note:** The **Allow specified actions and reusable workflows** option is only available in public repositories with the GitHub Free, GitHub Pro, GitHub Free for organizations, or GitHub Team plan.

Weird.

**Reply by brandonfl** [reply]
strange :thinking: 
Thanks for the information :smile: 

I have opened a support ticket and will share the result. Maybe a bug

**Reply by n3dst4** [reply]
@brandonfl Did you ever get an answer? Two years later I just ran into this myself!
